EEBUS panic on shutdown

### Describe the bug

I get a panic when terminating evcc:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x1581fac]

goroutine 1682 [running]:
github.com/evcc-io/evcc/meter.(*EEBus).maUseCaseSupportUpdate(0x7f74627950, {0x0, 0x0})
        github.com/evcc-io/evcc/meter/eebus_events.go:61 +0xbc
github.com/evcc-io/evcc/meter.(*EEBus).UseCaseEvent(0x7f74627950, {0x0?, 0x7f75556ea0?}, {0x0, 0x0}, {0x23488a4?, 0x2?})
        github.com/evcc-io/evcc/meter/eebus_events.go:40 +0xd8
github.com/evcc-io/evcc/server/eebus.(*EEBus).ucCallback(0x7f752706c0, {0x7f75556ea0, 0x28}, {0x2d33e00, 0x7f74d396e0}, {0x0, 0x0}, {0x23488a4, 0x1b})
        github.com/evcc-io/evcc/server/eebus/eebus.go:316 +0x1e0
github.com/enbility/eebus-go/usecases/usecase.(*UseCaseBase).removeDeviceFromAvailableEntityScenarios(0x7f7536b930, {0x2d33e00, 0x7f74d396e0})
        github.com/enbility/eebus-go@v0.7.1-0.20260520155652-11bc53084557/usecases/usecase/usecase.go:230 +0xa4
github.com/enbility/eebus-go/usecases/usecase.(*UseCaseBase).deviceOrEntityRemoved(0x0?, {{0x7f75556ea0, 0x28}, 0x0, 0x2, {0x2d33e00, 0x7f74d396e0}, {0x0, 0x0}, {0x0, ...}, ...})
        github.com/enbility/eebus-go@v0.7.1-0.20260520155652-11bc53084557/usecases/usecase/events.go:32 +0x44
github.com/enbility/eebus-go/usecases/usecase.(*UseCaseBase).HandleEvent(0x7f7536b930, {{0x7f75556ea0, 0x28}, 0x0, 0x2, {0x2d33e00, 0x7f74d396e0}, {0x0, 0x0}, {0x0, ...}, ...})
        github.com/enbility/eebus-go@v0.7.1-0.20260520155652-11bc53084557/usecases/usecase/events.go:13 +0x4c
created by github.com/enbility/spine-go/spine.(*events).Publish in goroutine 1338
        github.com/enbility/spine-go@v0.7.1-0.20260520153416-0104ce40c885/spine/events.go:108 +0x2ec
```

[eebus_panic(trace).log](https://github.com/user-attachments/files/29147255/eebus_panic.trace.log)

Only the controlbox has been connected via EEBUS.

### Steps to reproduce

1. Start evcc via command line
2. STRG+C to close

### What type of operating system or environment does evcc run on?

Linux

### External automation

- [x] I have made sure that no external automation like HomeAssistant or Node-RED is active or accessing any of the mentioned devices when this issue occurs.

### Nightly build

- [x] I have verified that the issue is reproducible with the latest nightly build

### Version

evcc version 0.309.2 (eed0649ea)

--- a/meter/eebus_events.go
+++ b/meter/eebus_events.go
@@ -34,7 +34,7 @@ func (c *EEBus) Connect(connected bool) {
 
 // UseCaseEvent implements the eebus.Device interface
 func (c *EEBus) UseCaseEvent(_ spineapi.DeviceRemoteInterface, entity spineapi.EntityRemoteInterface, event eebusapi.EventType) {
-	// device/entity removal fires support-update events with a nil entity
+	// deviceremoval fires support-update events with a nil entity
 	if entity == nil {
 		return
 	}

--- a/meter/eebus_events.go
+++ b/meter/eebus_events.go
@@ -34,6 +34,11 @@ func (c *EEBus) Connect(connected bool) {
 
 // UseCaseEvent implements the eebus.Device interface
 func (c *EEBus) UseCaseEvent(_ spineapi.DeviceRemoteInterface, entity spineapi.EntityRemoteInterface, event eebusapi.EventType) {
+	// device/entity removal fires support-update events with a nil entity
+	if entity == nil {
+		return
+	}
+
 	switch event {
 	// Monitoring Appliance
 	case mpc.UseCaseSupportUpdate, mgcp.UseCaseSupportUpdate:

